### ISSUE TITLE
[Magento2] Create plugin does not imports the class

### ISSUE THREAD

**Issue by Koc** [enriquepiatti/Magicento#265]
```php

use Magento\Framework\App\ResponseInterface;

    public function renderResult(ResponseInterface $response) // generate plugin for this class
```

got
```php
<?php

namespace Brouzie\Satin\Plugin\View\Result;

class LayoutPlugin
{

    public function aroundRenderResult(
        \Magento\Framework\View\Result\Layout $subject,
        callable $proceed,
        ResponseInterface $response
    ) {
    }
}
```

expexted FQCN `ResponseInterface` or imported class.

**Comment by Koc** [comment]
also when target class hasn't any argument - we got invalid code

```php
<?php

namespace Brouzie\Satin\Plugin\Model\Export;

class ProductPlugin
{


    public function aroundExport(\Magento\CatalogImportExport\Model\Export\Product $subject, callable $proceed}
```

**Comment by Koc** [comment]
seems like #245 returns

**Comment by enriquepiatti** [comment]
This should be fixed in v1.7.0. Please update.
